lv_conf_template.h

```diff
@@ -581,380 +625,404 @@
+/** 1: Enable system monitor component */
 #define LV_USE_SYSMON   0
 #if LV_USE_SYSMON
-    /*Get the idle percentage. E.g. uint32_t my_get_idle(void);*/
+    /** Get the idle percentage. E.g. uint32_t my_get_idle(void); */
     #define LV_SYSMON_GET_IDLE lv_timer_get_idle
 
-    /*1: Show CPU usage and FPS count
-     * Requires `LV_USE_SYSMON = 1`*/
+    /** 1: Show CPU usage and FPS count.
+     *  - Requires `LV_USE_SYSMON = 1` */
     #define LV_USE_PERF_MONITOR 0
     #if LV_USE_PERF_MONITOR
         #define LV_USE_PERF_MONITOR_POS LV_ALIGN_BOTTOM_RIGHT
 
-        /*0: Displays performance data on the screen, 1: Prints performance data using log.*/
+        /** 0: Displays performance data on the screen; 1: Prints performance data using log. */
         #define LV_USE_PERF_MONITOR_LOG_MODE 0
     #endif
 
-    /*1: Show the used memory and the memory fragmentation
-     * Requires `LV_USE_STDLIB_MALLOC = LV_STDLIB_BUILTIN`
-     * Requires `LV_USE_SYSMON = 1`*/
+    /** 1: Show used memory and memory fragmentation.
+     *     - Requires `LV_USE_STDLIB_MALLOC = LV_STDLIB_BUILTIN`
+     *     - Requires `LV_USE_SYSMON = 1`*/
     #define LV_USE_MEM_MONITOR 0
     #if LV_USE_MEM_MONITOR
         #define LV_USE_MEM_MONITOR_POS LV_ALIGN_BOTTOM_LEFT
     #endif
-
 #endif /*LV_USE_SYSMON*/
 
-/*1: Enable the runtime performance profiler*/
+/** 1: Enable runtime performance profiler */
 #define LV_USE_PROFILER 0
 #if LV_USE_PROFILER
-    /*1: Enable the built-in profiler*/
+    /** 1: Enable the built-in profiler */
     #define LV_USE_PROFILER_BUILTIN 1
     #if LV_USE_PROFILER_BUILTIN
-        /*Default profiler trace buffer size*/
-        #define LV_PROFILER_BUILTIN_BUF_SIZE (16 * 1024)     /*[bytes]*/
+        /** Default profiler trace buffer size */
+        #define LV_PROFILER_BUILTIN_BUF_SIZE (16 * 1024)     /**< [bytes] */
     #endif
 
-    /*Header to include for the profiler*/
+    /** Header to include for profiler */
     #define LV_PROFILER_INCLUDE "lvgl/src/misc/lv_profiler_builtin.h"
 
-    /*Profiler start point function*/
+    /** Profiler start point function */
     #define LV_PROFILER_BEGIN    LV_PROFILER_BUILTIN_BEGIN
 
-    /*Profiler end point function*/
+    /** Profiler end point function */
     #define LV_PROFILER_END      LV_PROFILER_BUILTIN_END
 
-    /*Profiler start point function with custom tag*/
+    /** Profiler start point function with custom tag */
     #define LV_PROFILER_BEGIN_TAG LV_PROFILER_BUILTIN_BEGIN_TAG
 
-    /*Profiler end point function with custom tag*/
+    /** Profiler end point function with custom tag */
     #define LV_PROFILER_END_TAG   LV_PROFILER_BUILTIN_END_TAG
 #endif
 
-/*1: Enable Monkey test*/
+/** 1: Enable Monkey test */
 #define LV_USE_MONKEY 0
 
-/*1: Enable grid navigation*/
+/** 1: Enable grid navigation */
 #define LV_USE_GRIDNAV 0
 
-/*1: Enable lv_obj fragment*/
+/** 1: Enable `lv_obj` fragment logic (@see  https://docs.lvgl.io/master/others/fragment.html ) */
```

Comment:
Why did you add a link only here?

Reply:
Good question.  I hadn't heard of `LV_USE_FRAGMENT` before so I took the time to look up in the documentation, and I thought it would be appropriate to bring helpful knowledge back into `lv_conf.h`.  I thought of it as "enhancement by increments".  Are you thinking like I am -- that it would be a good idea to provide links for all of them?

Reply:
Now the docs main section titles mostly mimics the structure of `lv_conf.h`
- "WIDGETS"
- "OTHERS"
- "3RD PARTY LIBS"

If you really find it useful, I'd add links below the main sections. E.g
```
/*==================
 * OTHERS
 *==================*/
/* @see  https://docs.lvgl.io/master/others/index.html */
```

Reply:
Hi, Gabor!  ( @kisvegabor )

I did find it useful, and think that idea will be helpful for others as well.  So following your suggestion, I added documentation links (following the nice example already under WIDGETS) to the major sections below the WIDGETS section:  THEMES, LAYOUTS, 3RD PARTY LIBRARIES and OTHERS.  I did not find a clear connection between the DEVICES section and the documentation, so I left it alone.

Given that the `@see` is only useful in a Doxygen comment that documents a code member (or file or directory), I replaced it with the clear example already under WIDGETS section heading.  So they look like this:

```
/* Documentation for ______ can be found here: https://docs.lvgl.io/master/others/index.html . */
```

Submitted and ready again for review.